src/services/auth/utils.ts

```diff
@@ -14,6 +14,9 @@ import {
 import { InvalidSession, OrphanSession } from '../../utils/errors';
 import { MemberRepository } from '../member/repository';
 
+// todo: duplicate?
+const memberRepository = new MemberRepository();
```

Comment:
What if we wants to use a manager with our utils functions ? Is it possible to provide the repository as a parameter of our function, so we don't instantiate it here ?